app.rb

```diff
@@ -0,0 +1,87 @@
+require "sinatra/base"
+require "sinatra/reloader"
+require "./lib/database_connection"
+require_relative "lib/peep_repository"
+require_relative "lib/user_repository"
+require "date"
+require "bcrypt"
+
+DatabaseConnection.connect("chitter_test")
```

Comment:
Did you have any issues with connecting to the database? I took (what I think is) the same approach as you in using the template for database connection (including in lib/database_connection) that we used in previous weeks, but this was causing me a problem where test comments were actually being added to my database when I ran rspec - so I looked back at the web-applications music-library-app and copied all the database_connection info from there and this solved the problem. I'd recommend taking a look at that too (or my lib/database_connection  file and 'DatabaseConnection.connect' line in my app.rb) for future reference if you encounter similar problems